lib/rummager/app.rb

```diff
@@ -343,6 +344,27 @@ def get_type_from_request_body(request_body)
     GovukHealthcheck.healthcheck(checks).to_json
   end
 
+  get "/sitemap.xml" do
+    serve_from_s3("sitemap.xml")
+  end
+
+  get "/sitemaps/:sitemap" do |sitemap|
+    serve_from_s3("sitemaps/#{sitemap}")
+  end
+
+  def serve_from_s3(key)
+    o = Aws::S3::Object.new(bucket_name: ENV["AWS_S3_BUCKET_NAME"], key: key)
```

Comment:
Do you need to guard against `AWS_S3_BUCKET_NAME` not being defined?

Reply:
`Aws::S3::Object.new` will throw an error about a missing bucket name if it's `nil`.